adafruit_circuitplayground/circuit_playground_base.py

```diff
@@ -184,6 +184,58 @@ def detect_taps(self, value):
                     value, 60, time_limit=10, time_latency=50, time_window=255
                 )
 
+    # pylint: disable-msg=too-many-arguments
+    # Many default arguments which will probably not need changing in most cases
+    def configure_taps(
+        self,
+        value,
+        accel_range=adafruit_lis3dh.RANGE_8_G,
+        theshold=None,
+        time_limit=None,
+        time_latency=50,
+        time_window=255,
+    ):
+        """ A way to customize the tap detection better. The default values don't work
+            for all cases.
+            Higher default thresholds for the CPB
+        """
```

Comment:
One of the most important parts of this library is the in-line documentation which is rendered by Sphinx, hosted on Read the Docs, and makes up the API documentation. You can use the many other docstrings as examples of what we're looking for. I'll give a quick explanation as well.

The beginning of the docstring should simply explain what the function does. You don't have to justify your addition in the docstring, simply explain what it is. For example, something to the effect of:
```
"""Granular configuration of tap parameters."""
```
You can come up with something better if you like!

Then please include each parameter and a quick sentence explaining it - you can crib from [the LIS3DH library](https://github.com/adafruit/Adafruit_CircuitPython_LIS3DH/blob/bea5d4a347aeece71a421ee292551264e3bf7ae2/adafruit_lis3dh.py#L268) since this function is basically exposing `set_tap`. 

Following that, please include a `code-block` with an example of how to use it. I struggled to figure out how to use it, and as it's not included in an example (which is fine), having an example in the API documentation is crucial.

Reply:
More documentation to come with the next commit.